setup.py

```diff
@@ -0,0 +1,66 @@
+import re
+import subprocess
+from io import open
+
+from setuptools import find_packages, setup
+
+
+def fetch_requirements(path):
+    with open(path, "r") as fd:
+        return [r.strip() for r in fd.readlines()]
+
+
+def is_commit_on_tag():
+    try:
+        result = subprocess.run(
+            ["git", "describe", "--exact-match", "--tags"], capture_output=True, text=True, check=True
+        )
+        tag_name = result.stdout.strip()
+        return tag_name
+    except subprocess.CalledProcessError:
+        return False
+
+
+def get_build_version():
+    if is_commit_on_tag():
+        return __version__
+    try:
+        result = subprocess.run(["git", "describe", "--tags"], capture_output=True, text=True, check=True)
+        _, distance, commit = result.stdout.strip().split("-")
+        return f"{__version__}.dev{distance}+{commit}"
+    except subprocess.CalledProcessError:
+        return __version__
+
+
+try:
+    filepath = "./neural_compressor_ort/version.py"
+    with open(filepath) as version_file:
+        (__version__,) = re.findall('__version__ = "(.*)"', version_file.read())
+except Exception as error:
+    assert False, "Error: Could not open '%s' due %s\n" % (filepath, error)
+
+
+if __name__ == "__main__":
+
+    setup(
+        name="neural_compressor_ort",
```

Comment:
Is there any plans to release an onnx-neural-compressor package?

Reply:
I believe the binary release is required, though this needs to be discussed in SIG/committee.

Reply:
I think the suggestion is to drop "_ort". Justin can you clarify if that's what your feedback was pointing to?

Reply:
I understand neural-compressor is an existing package that is not ONNX specific. For this ONNX specific package I wonder if names like onnx-neural-compressor would make sense. I think it is important that it maintains compatibility with ORT, but does not target ORT solely, as the current name would suggest.

Reply:
we renamed binary name to onnx-neural-compressor.  To be consistent with common practice, we plan to rename source folder name from neural_compressor_ort to onnx_neural_compressor.  

user will install ONC like:
pip install onnx-neural-compressor

user will import like:
import onnx_neural_compressor.quantization

@justinchuby @freddychiu,  if there is no concern, we will proceed like this